src/data/faq.json

```diff
@@ -623,7 +623,19 @@
             {
                 "title": "Risk Assessment",
                 "id": "risk_assessment",
-                "accordion": [
+                "accordion": [
+                    {
+                        "title": "Dissimilar indication of risk status",
+                        "anchor": "dissimilar_indication_of_risk_status",
+                        "active": true,
+                        "textblock": [
+                            "Why do differences in infection risk status appear in different CWA app's, even for close-by smartphones?",
+                            "Varying infection risk ratings are caused by several reasons. However, they do not affect the effectiveness of the Corona Warning App (CWA).",
```

Comment:
Corona Warning App => Corona-Warn-App
(Use the exact name as it appears in the app itself when the mobile device language is set to English.)

Reply:
ok !